pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -360,7 +360,7 @@ func (c *Controller) remainingTimeToRetry(pr *autoscaling.ProvisioningRequest, f
 }
 
 func (c *Controller) handleError(ctx context.Context, wl *kueue.Workload, ac *kueue.AdmissionCheckState, msg string, err error) error {
-	ac.Message = msg
+	ac.Message = api.TruncateConditionMessage(msg)
```

Comment:
Please also make sure the full message is logged. 

Also, to make the code easier (avoid duplication) I would prefer to send the event from within `handleError`. Then, we don't need to pass both `msg` and `err` as parameters (`err` would be enough).

Reply:
> Also, to make the code easier (avoid duplication) I would prefer to send the event from within handleError. Then, we don't need to pass both msg and err as parameters (err would be enough).

Done